account.go

```diff
@@ -315,6 +315,53 @@ func TxPayments(txID string) ([]horizon.Payment, error) {
 	return page.Embedded.Records, nil
 }
 
+// PaymentsPlus contains the payments in a transaction as well as the memo.
+type PaymentsPlus struct {
+	TransactionID string
+	Payments      []horizon.Payment
+	Memo          string
+	MemoType      string
+	FeePaid       int32
+}
+
+// TxPaymentsPlus gets the payment operations in a transaction as well as
+// some details about the transaction.
+func TxPaymentsPlus(txID string) (*PaymentsPlus, error) {
```

Comment:
Seems like just the addition of TxDetails is enough. It has more fields and it's more transparent how many underlying RPCs you're hitting.